rclpy/rclpy/validate_topic_name.py

```diff
@@ -16,6 +16,8 @@
 from rclpy.exceptions import InvalidTopicNameException
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 
+TOPIC_SEPARATOR_STRING = '/'
```

Comment:
I tried to find a "sensible" place for the topic separator constant because it seems like it could be used for more than just parameter services and this was the best I could come up with based on module names. But I can either put it in parameter_services for now or if there's a place more "sensible" than this I can move it to I'm happy to field suggestions.